blueprints/gcve/monitoring/README.md

```diff
@@ -0,0 +1,93 @@
+# Google Cloud VMWare Engine Logging Monitoring Module
+
+The Blueprinte sets up Monitoring and Syslog Logging for Google Cloud VMWare Engine Private Clouds.
+
+Infrastructure monitoring and logging for GCVE is commonly set up using a [standalone Bindplane agent](https://cloud.google.com/vmware-engine/docs/environment/howto-cloud-monitoring-standalone). This module deploys the bindplane agent using a Managed Instance Group to accept metrics and Syslog logs from VMware vCenter and forward them to Cloud Monitoring and Cloud Logging. To forward syslog messages, please refer to the following documentation on [how to configure a private cloud for syslog forwarding](https://cloud.google.com/vmware-engine/docs/environment/howto-forward-syslog).
```

Comment:
Bindplane/bindplane

Reply:
Syslog/syslog (in this case better use lowercase, it's not a product)

Reply:
Bindplane consistently capitalized, syslog in lowercase.